README.md

```diff
@@ -5,16 +5,35 @@ Those credentials will be valid for 12 hours by default.
 
 ## Getting started
 
+### Install using go...
+
+For this, go must be installed on your system. 
+
 Install executable with golang
 ```bash
-go get [-u] github.com/signavio/aws-mfa-login
+go get github.com/signavio/aws-mfa-login
 ```
-or download from releases
-```yaml
+.. or update by setting the -u flag: 
+```bash
+go get -u github.com/signavio/aws-mfa-login
+``` 
+
+Make sure your go path is part of your PATH environment variable: 
+```
+export PATH="${PATH}:~/go/bin/"
+```
+
+### .. or download from releases
+
+```console
 curl -L https://github.com/signavio/aws-mfa-login/releases/latest/download/aws-mfa-login_linux_amd64.gz -o aws-mfa-login.gz
```

Comment:
Could you make me favor and replace Linux with $(uname) with will work for Mac and Linux

Reply:
ok, done